docs/plugins/ingest-useragent.asciidoc

```diff
@@ -0,0 +1,66 @@
+[[ingest-useragent]]
+=== Ingest Useragent Processor Plugin
+
+The Useragent processor extracts details from the useragent string a browser sends with its web requests. 
```

Comment:
+1 I'll open a PR and change the naming.

(I missed this conversation as it was hidden when the pr was updated)